core/src/Instance/B2CAuthority.cs

```diff
@@ -35,7 +35,7 @@ internal class B2CAuthority : AadAuthority
     {
         public const string Prefix = "tfp"; // The http path of B2C authority looks like "/tfp/<your_tenant_name>/..."
 
-        public B2CAuthority(string authority, bool validateAuthority) : base(authority, validateAuthority)
+        internal B2CAuthority(string authority, bool validateAuthority) : base(authority, validateAuthority)
```

Comment:
a public method in an internal class has the same visibility as an internal method in an internal class, so there is no reason to change this.

Reply:
public does not reflect reality even though compiler is fine with that